src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -0,0 +1,114 @@
+package org.edx.jenkins.dsl
+
+class AnalyticsConstants {
+
+    public static def common_log_rotator = { allVars ->
+        return {
+            daysToKeep(30)
```

Comment:
FYI, I'm making this closure configurable in my PR: https://github.com/edx/jenkins-job-dsl/pull/422/files#diff-4bbef760511c0e8e8f5a2d4a527bf72c